src/components/ClusterManager.js

```diff
@@ -363,6 +365,22 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  componentDidUpdate(prevProps) {
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const { status, jupyterUserScriptUri } = this.getCurrentCluster() || {}
```

Comment:
There was a recent change to the cluster manager so that it ignores Error clusters (not sure if it was intentional), so this logic doesn't currently work.